include/swift/Frontend/FrontendOptions.h

```diff
@@ -125,6 +125,7 @@ class FrontendOptions {
   /// Include local definitions/references in the index data.
   bool IndexIncludeLocals = false;
 
+  bool SerializeDebugInfo = false;
```

Comment:
I think this should get a more specific name: SerializeDebugInfoInSIL or something like that.